modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/asset/model/LayoutAssetRendererFactory.java

```diff
@@ -63,15 +63,19 @@ public AssetEntry getAssetEntry(String className, long classPK)
 
 		Layout layout = _layoutLocalService.getLayout(classPK);
 
-		User user = _userLocalService.getUserById(layout.getUserId());
-
 		AssetEntry assetEntry = _assetEntryLocalService.createAssetEntry(
 			classPK);
 
 		assetEntry.setGroupId(layout.getGroupId());
-		assetEntry.setCompanyId(user.getCompanyId());
-		assetEntry.setUserId(user.getUserId());
-		assetEntry.setUserName(user.getFullName());
+
+		User user = _userLocalService.fetchUser(layout.getUserId());
+
+		if (user != null) {
+			assetEntry.setCompanyId(user.getCompanyId());
+			assetEntry.setUserId(user.getUserId());
+			assetEntry.setUserName(user.getFullName());
+		}
+
```

Comment:
This looks wrong. If the user doesn't exist, the assetEntry will be partially updated. If the user is null, you should do something like this:
```
if (user != null) {
    ...
}
else {
    assetEntry.setCompanyId(layout.getCompanyId());
    assetEntry.setUserId(_userLocalService.getDefaultUserId(layout.getCompanyId());
    ...
}
```